sagemathinc/cocalc
# Fix hooks in share server

# Description
Fix #4284

# Testing Steps
1. Start the share server
1. Open a jupyter notebook
1. It should appear

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
